docs/sources/setup/install/helm/install-microservices/_index.md

```diff
@@ -176,48 +181,57 @@ When deploying Loki using S3 Storage **DO NOT** use the default bucket names;  `
 ```s3
 # Example configuration for Loki with S3 storage
 
-  loki:
-    schemaConfig:
-      configs:
-        - from: 2024-04-01
-          store: tsdb
-          object_store: s3
-          schema: v13
-          index:
-            prefix: loki_index_
-            period: 24h
-    ingester:
+loki:
+  schemaConfig:
+    configs:
+      - from: 2024-04-01
+        store: tsdb
+        object_store: s3
+        schema: v13
+        index:
+          prefix: loki_index_
+          period: 24h
+  storage_config:
+    aws:
+      region: <AWS region your bucket is in eg. `eu-west-2`>
```

Comment:
Change to 

`region: eu-west-2. # Change to suit your AWS region`

for parallel structure with next line